.github/dependabot.yml

```diff
@@ -18,6 +18,28 @@ updates:
     schedule:
       interval: "daily"
     groups:
-      pip:
+      action:
```

Comment:
the group name `pip` is correct. The `action` group is not descriptive of the `uv` ecosystem.